### ISSUE TITLE
Output images are completely black

### ISSUE THREAD

**Issue by MFarooqAit** [jtiger958/ESRGAN-pytorch#2]
Dear author,
Thank you so much for sharing the simplified code of ESRGAN. 

I have used your training code for my own dataset, but testing code produced completely black super resolved images.

Kindly guide. 
Best,
Farooq

**Comment by purijs** [comment]
Same issue, were you able to resolve it?

**Comment by MFarooqAit** [comment]
Dear purijis,
Try to load your trained model by editing the test.py file, e.g,
net.load_state_dict(torch.load('checkpoints/generator_100.pth'))

Results are still not very impressive.

**Comment by purijs** [comment]
> Dear purijis,
> Try to load your trained model by editing the test.py file, e.g,
> net.load_state_dict(torch.load('checkpoints/generator_100.pth'))
> 
> Results are still not very impressive.

Thanks, also the loss values keep fluctuating for me, is it happening for you too? I tried training on 100 images for 500 epochs

**Comment by MFarooqAit** [comment]
I did not observed it, but results are not very good on my dataset (1600 images) trained for 200 epochs.

**Comment by purijs** [comment]
> I did not observed it, but results are not very good on my dataset (1600 images) trained for 200 epochs.

Right, for me, model is also not generalising well for test data, what other approaches or gan models are you trying?

**Comment by thunanguyen** [comment]
I think we need to train around 100k epochs to have a good GAN guys

**Comment by purijs** [comment]
> I think we need to train around 100k to have a good GAN guys

100K of what, epochs or images?

**Comment by thunanguyen** [comment]
Sr, I've edited to make it more clear

Vào Th 4, 4 thg 11, 2020 vào lúc 16:37 Jaskaran Singh Puri <
notifications@github.com> đã viết:

**Comment by veb-101** [comment]
Any updates on this?
